src/KafkaFlow.Retry.SqlServer/Deploy/01 - Create_Tables.sql

```diff
@@ -130,6 +130,41 @@ BEGIN
 END
 GO
 
+IF(NOT EXISTS(SELECT *
+              FROM sys.types
+              WHERE is_table_type=1 AND name='TY_RetryQueueItemsIds'))
+BEGIN
+    CREATE TYPE [dbo].[TY_RetryQueueItemsIds]
```

Comment:
if we change the stored procedure to `spProcedureName`, maybe we can also set the type with equivalent name like `typeRetryQueueItemsIds` or `tyRetryQueueItemsIds`